scipy/stats/_stats_py.py

```diff
@@ -3477,6 +3522,13 @@ def median_abs_deviation(x, axis=0, center=np.median, scale=1.0,
         output data-type is ``np.float64``. Otherwise, the output data-type is
         the same as that of the input.
 
+    Raises
+    ------
+    TypeError
+        If `center` is not callable.
```

Comment:
Not desirable. If it's an argument document as `callable`, and you pass in the wrong type then `TypeError` is the standard expectation in Python code.